malicious.ts

```diff
@@ -0,0 +1,10 @@
+// Example with OWASP Top 10 Violations (XSS and SQL Injection)
+
+// Vulnerability 1: Cross-Site Scripting (XSS)
+const userInput = "<script>alert('XSS Attack!');</script>";
+document.getElementById("output").innerHTML = userInput; // This is vulnerable to XSS
```

Comment:
>_vulnerability_ 10

**✨ AI Suggestion:** Direct use of user-provided input in `innerHTML` is a critical Cross-Site Scripting (XSS) vulnerability.  Malicious scripts injected via user input can execute in the user's browser, potentially stealing their session, redirecting them to phishing sites, or defacing the application.
```suggestion
document.getElementById("output").textContent = userInput;
```